core/src/main/java/com/graphhopper/reader/osm/OSMTurnRestriction.java

```diff
@@ -26,23 +26,29 @@
  */
 public class OSMTurnRestriction {
     private final long fromOsmWayId;
-    private final long viaOsmNodeId;
+    private final ArrayList<Long> viaOSMIds;
```

Comment:
The ArrayList is just an implementation detail. Stick with `List` for fields and method arguments.

Reply:
fixed